setup.py

```diff
@@ -107,5 +107,5 @@ def find_rel_test():
     provides=["eb"] + easybuild_packages,
     test_suite="test.framework.suite",
     zip_safe=False,
-    install_requires=["vsc-base >= 2.2.4"],
+    install_requires=["vsc-base >= 2.2.4", "PyYAML >= 3.11"],
```

Comment:
for now, we should probably make `PyYAML` an optional dependency (see http://pythonhosted.org/setuptools/setuptools.html#declaring-extras-optional-features-with-their-own-dependencies)

that allows us to include whatever support we have for `.yeb` in the next release under `--experimental`, if it can be useful to let people start playing around with it and provide feedback